sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -415,7 +424,7 @@ def process(self, element, schema_mod_job_name_prefix):
     uid = _bq_uuid()
     job_name = '%s_%s_%s' % (schema_mod_job_name_prefix, destination_hash, uid)
 
-    _LOGGER.debug(
+    _LOGGER.info(
```

Comment:
Schema modification job doesn't often happen 1-2 per job. Adding as info level will help in troubleshooting.

Reply:
SGTM